notifications/README.md

```diff
@@ -0,0 +1,58 @@
+# Notifications
+
+## Notification Object
+
+```
+{
+  id: string,
+  username: string,
+  message: string,
+  createOn: unix-timestamp,
+  url: string,
+  type: string
+}
+```
+
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /notifications](#get-notifications)      | Returns notifications of the logged in user |
+
+## **GET /notifications**
+
+Returns all notifications for a logged in user
+
+- **Params**  
+  None
+- **Query**  
+  page=[integer], n=[integer]
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'Notifications returned successfully!',
+  notifications: [
+    <notification-object>,
+    <notification-object>,
+    <notification-object>,
+    <notification-object>,
+    <notification-object>
+  ]
+}
```

Comment:
Should we also return pagination details here?

Reply:
Didn't exactly get you..?

Reply:
I meant, if the API is returning responses, is it not telling us which `page` and which `n` was these set of notifications for?
That may help is in loading the next set, e.g when a user scrolls down further